README.md

```diff
@@ -87,3 +95,31 @@ git checkout main
 git pull
 pnpm run deploy
 ```
+
+## Further Reading
+Sub-projects have their own READMEs with more in-depth information.
+
+[AntAlamanc](apps/antalmanac)
```

Comment:
`apps/antalmanac` doesn't actually have it's own README

Reply:
Ok, I'll just move the backend README section and nuke the frontend README link.